db/src/main/resources/db/migration/V1_003__legacy_datafeed_initial_format.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE dwp_legacy_household (
+    id uuid not null primary key,
+    household_identifier varchar(50),
+    file_import_number integer,
+    award varchar(500),
```

Comment:
What is the `award` column for?

Reply:
I believe this is the type of benefit they're recieving, but as I wasn't completely sure I left it with the wording from the document